R/serializers.R

```diff
@@ -1,3 +1,21 @@
+#' Add a function for serializing an input before bookmarking application state
+#'
+#' @param inputId Name of the input value.
+#' @param fun A function that takes the input value and returns a modified
+#'   value. The returned value will be used for the test snapshot.
+#' @param session A Shiny session object.
```

Comment:
Could this use an example? Or are you thinking this is so advanced that nobody is really going to need it? (If so maybe `@keyword internal` or whatever that thing is?)